src/core/src/core_logic/PatchInstaller.py

```diff
@@ -148,6 +160,64 @@ def raise_if_min_python_version_not_met(self):
+            - Only security updates (no other configuration) - simplistic execution flow; no advanced evaluation is desired or necessary.
+        """
+        installed_update_count = 0  # includes dependencies
+        patch_installation_successful = True
+        maintenance_window_exceeded = False
+        remaining_time = maintenance_window.get_remaining_time_in_minutes()
+
+        try:
+            all_packages, all_package_versions = package_manager.get_all_updates(True)
+            packages, package_versions = package_manager.get_security_updates()
+            self.last_still_needed_packages = list(all_packages)
+            self.last_still_needed_package_versions = list(all_package_versions)
+
+            not_included_packages, not_included_package_versions = self.get_not_included_updates(package_manager, packages)
+            packages, package_versions, self.skipped_esm_packages, self.skipped_esm_package_versions, self.esm_packages_found_without_attach = package_manager.separate_out_esm_packages(packages, package_versions)
+
+            self.status_handler.set_package_install_status(not_included_packages, not_included_package_versions, Constants.NOT_SELECTED)
+            self.status_handler.set_package_install_status(packages, package_versions, Constants.PENDING)
+            self.status_handler.set_package_install_status(self.skipped_esm_packages, self.skipped_esm_package_versions, Constants.FAILED)
+
+            self.status_handler.set_package_install_status_classification(packages, package_versions, classification="Security")
+            package_manager.set_security_esm_package_status(Constants.INSTALLATION, packages)
+
+            installed_update_count = 0  # includes dependencies
+            patch_installation_successful = True
+            maintenance_window_exceeded = False
+
+            install_result = Constants.FAILED
+            for i in range(0, Constants.MAX_INSTALLATION_RETRY_COUNT):
+                code, out = package_manager.install_security_updates_azgps_coordinated()
+                installed_update_count += self.perform_status_reconciliation_conditionally(package_manager)
+
+                remaining_time = maintenance_window.get_remaining_time_in_minutes()
+                if remaining_time < 120:
+                    raise Exception("Not enough safety-buffer to continue strict safe deployment.")
+
+                if code != 0:   # will need to be modified for other package managers
+                    if i < Constants.MAX_INSTALLATION_RETRY_COUNT - 1:
+                        time.sleep(i * 5)
+                        self.composite_logger.log_warning("[PI][AzGPS-Coordinated] Non-zero return. Retrying. [RetryCount={0}][TimeRemainingInMins={1}][Code={2}][Output={3}]".format(str(i), str(remaining_time), str(code), out))
+                else:
+                    patch_installation_successful = True
+                    break
+        except Exception as error:
```

Comment:
Lines 210 - 211 marked not covered by UTs

Reply:
Reviewed with breakpoints - coverage should show up as iteration evaluation finishes.